### DISCUSSION TITLE
ortoni-report is trying to open in Jenkins automatically

### DISCUSSION THREAD

**Discussion by Raja442014** [ortoniKC/ortoni-report#51] [Q&A]
Eventhough - we have given open: never or open: process.env.CI ? "never" : "always",
Its always trying to open in jenkins and making the stage in run state. Can you please share examples of jenkins and playwright config file ?

**Comment by ortoniKC** [comment]
In jenkins you have to pass CI = true as an env variable.

**Reply by Raja442014** [reply]
yeah - I made it as CI - true, 
its still trying to open the report on the Jenkins.
FYI..am using '--reporter=ortoni-report' and its try to open the report always and never comesout of the stage.
Also - stage is always PASS - even though they are failed tests in the report.

**Comment by ortoniKC** [comment]
The default is set to never, so it wont open in jenkins by default.
Fixed in V3.0.0
